README.rst

```diff
@@ -11,13 +11,34 @@ Currently MorphTool provides:
 
 Installation
 ------------
+It is recommended to install in a fresh virtualenv.
 
-In a fresh virtualenv:
+Base installation:
 
 .. code:: bash
 
     pip install morph-tool
 
+If plan to use `dendrogram` module:
+
+.. code:: bash
+
+    pip install morph-tool[plot]
+
+If plan to use `nrnhines` module:
+
+.. code:: bash
+
+    pip install morph-tool[nrn]
+
+If plan to convert folders of morphologies:
```

Comment:
does the folder converter work single process? or is dask a requirement?

Reply:
`dask` is requirement but I wonder how it works because only `dask.bag` is used. I see no `Client` usage from `dask`.

Reply:
tbh, multiprocssing should work fine for that, if you want to remove dask dependency.

Reply:
don't step back from the righteous way of dask

Reply:
I wouldn't dare! haha